src/compiler/diagnosticMessages.json

```diff
@@ -3118,7 +3118,7 @@
         "category": "Error",
         "code": 18000
     },
-    "The path in an 'extends' options must be relative or rooted.": {
+    "The paths in an 'extends' option must be relative or rooted.": {

```

Comment:
@weswigham yeah you are right, I was reading the original proposal in which it can be either string or array. It should just be `path` here.